src/JsonRpc/Client.php

```diff
@@ -21,6 +21,7 @@ public function __construct(string $baseUri, private string $service = 'object')
                 'Content-Type' => 'application/json',
             ],
             'base_uri' => $baseUri,
+            'verify' => config('odoo.ssl_verify',true),
```

Comment:
Use a Constructor param instead of the config. move the config call to the laravel OdooServiceProvider.php

Reply:
i have made revision on b5785407f0fce587c26876d6dcee884d90341ab5